lib/ClangImporter/ImportDecl.cpp

```diff
@@ -3040,6 +3041,8 @@ namespace {
       return result;
     }
 
+    using ProtocolSet = llvm::SmallSet<ProtocolDecl *, 4>;
```

Comment:
Since this is a `Set`, I wonder if the non-deterministic order of protocols would bite us later. Looks like the compiler might start emitting diagnostics in a different order. These kind of things are usually harmless to users, but make testing harder. Could we switch to a `SmallVector` here to preserve the order here?

Reply:
Ah, I see, thanks!